engine/Default/alliance_leave_processing.php

```diff
@@ -2,27 +2,33 @@
 $action = $var['action'];
 if ($action == 'YES') {
 	$alliance =& $player->getAlliance();
-	// will this alliance be empty if we leave? (means one member right now)
+
 	if ($player->isAllianceLeader() && $alliance->getNumMembers() > 1) {
 		create_error('You are the leader! You must hand over leadership first!');
 	}
+
+	// will this alliance be empty if we leave? (means one member right now)
+	if ($alliance->getNumMembers() == 1) {
+		//$db->query('DELETE FROM alliance WHERE alliance_id = ' . $db->escapeNumber($player->getAllianceID()) . ' AND game_id = ' . $db->escapeNumber($player->getGameID()));
+		$db->query('DELETE FROM alliance_bank_transactions
```

Comment:
I wonder if this is something we actually want to delete, do admins use this to check multi/suspicious behaviour?